### PR TITLE
Docker compose (1 of 3)

### PR DESCRIPTION
**Pull request by jmarcet**
Maintainer: Javier Marcet <javier@marcet.info>
Compile tested: x86-64 master
Run tested: x86-64 master
Description:

First part of docker-compose, packages directly required by the former.

This comes from splitting #11155

### CODE REVIEW

File: lang/python/python-texttable/Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-texttable
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=module for creating simple ASCII tables
```
Comment (jefferyto) [inline lang/python/python-texttable/Makefile]:
```suggestion
  TITLE:=Module for creating simple ASCII tables
```

File: lang/python/python-texttable/Makefile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  URL:=https://github.com/foutaise/texttable/
+  DEPENDS:=+python3-light +python3-setuptools
```
Comment (jefferyto) [inline lang/python/python-texttable/Makefile]: Where is setuptools used during run-time?

File: lang/python/python-websocket-client/Makefile [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-websocket-client
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=WebSocket client for Python. hybi13 is supported
+  URL:=https://github.com/websocket-client/websocket-client
+  DEPENDS:=+python3-light +python3-setuptools +python3-six
```
Comment (jefferyto) [inline lang/python/python-websocket-client/Makefile]: Where is setuptools used during run-time?

Also, I believe python3-openssl (for [hashlib](https://github.com/websocket-client/websocket-client/blob/v0.57.0/websocket/_handshake.py#L22) and [ssl](https://github.com/websocket-client/websocket-client/blob/v0.57.0/bin/wsdump.py#L8)) and [python3-logging](https://github.com/websocket-client/websocket-client/blob/v0.57.0/websocket/_logging.py#L22) will be necessary.

File: lang/python/python-websocket-client/Makefile [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  VARIANT:=python3
+endef
+
+define Package/python3-websocket-client/description
+  WebSocket client for Python. hybi13 is supported
```
Comment (jefferyto) [inline lang/python/python-websocket-client/Makefile]:
```suggestion
  websocket-client module is WebSocket client for python. This provide the low level APIs for WebSocket. All APIs are the synchronous functions.

  websocket-client supports only hybi-13.
```
(wrap lines as appropriate)

File: lang/python/python-texttable/Makefile [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-texttable
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Module for creating simple ASCII tables
+  URL:=https://github.com/foutaise/texttable/
+  DEPENDS:=+python3-light
```
Comment (jefferyto) [inline lang/python/python-texttable/Makefile]: I think you will also need python3-codecs (for [unicodedata](https://github.com/foutaise/texttable/blob/v1.6.2/texttable.py#L94)).

**Comment by jefferyto** [comment]
ping @jmarcet

**Comment by jmarcet** [comment]
Sorry for the delay. I didn't notice any of this earlier.

I pushed the updated code already.

**Comment by jmarcet** [comment]
I'm really sorry for the delay.

I just pushed a rebased branch without the .swp file.

**Pull request merged**